# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: ipc/glue/BackgroundParentImpl.h
# Commit: dbdca4661a35
# Full Hash: dbdca4661a35c55d343cb76fd10366cdbd1e4c09
# Author: Eden Chuang <echuang@mozilla.com>
# Date: 2023-01-18 09:51:31
# Regressor Bug: 1351231
# File Overlap Count: 2
# Description:
#   Bug 1351231 - PFetch protocol declaration and implementation. r=dom-worker-reviewers,jesup
#   
#   Depends on D138813
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D142436
# ==============================================================================

diff -r ea27cd66fefd -r dbdca4661a35 ipc/glue/BackgroundParentImpl.h
--- a/ipc/glue/BackgroundParentImpl.h	Tue Jan 17 15:13:44 2023 +0000
+++ b/ipc/glue/BackgroundParentImpl.h	Tue Jan 17 15:13:45 2023 +0000
@@ -415,6 +415,8 @@
 
   already_AddRefed<PLockManagerParent> AllocPLockManagerParent(
       const ContentPrincipalInfo& aPrincipalInfo, const nsID& aClientId) final;
+
+  already_AddRefed<PFetchParent> AllocPFetchParent() override;
 };
 
 }  // namespace mozilla::ipc